### DISCUSSION TITLE
XRechnung per PDF

### DISCUSSION THREAD

**Discussion by JohannMaierhofer** [openjverein/jverein#321] [Ideas]
Ich habe gesehen, dass Hibiscus 2.10.21 den Import von XRechnung per PDF unterstützt.
Wäre es da nicht sinnvoll wenn JVerein eine solche Rechnung erzeugen könnte. Also z.B. als Standard Rechnung ohne Vorlage, ähnlich wie bei der Standard Spendenbescheinigungen.
Wenn ein Mitglied auch Hibiscus verwendet, könnte es die Rechnung dann gleich auch importieren.

Ich bin da wohl noch etwas überfordert mit so einer Implementierung. Aber vielleicht gibt es ja jemanden der sich damit besser auskennt und Lust und Zeit hat das zu implementieren.

**Comment by lenilsas** [comment]
XRechnungen sind bald für B2B Rechnungen pflicht. Da die meisten Vereine wohl keine Rechnungen an andere Unternehmen sonder eher an die Mitglieder stellen sehe ich hier keinen großen Handlungsbedarf.
Gerne kann das jemand umsetzen, soweit ich gesehen habe ist das aber mit einem größeren Aufwand verbunden.

**Reply by mikeharris2017** [reply]
Wir stellen durchaus Rechnungen für Firmen im Rahmen von Anzeigen in der Vereinszeitung und bei Vermietung für Firmenfeiern. 
Aktuell setze ich aber darauf, dass wir weiterhin schnöde PDFs senden dürfen

**Reply by JohannMaierhofer** [reply]
Mit #454 werden wir E-Rechnungen (ZUGFeRD) erstellen koönnen. Das sind PDF mit eingebettetem XML. Das sollte dann für diesen Anwendungsfall auch in Zukunft ok sein.

**Reply by mikeharris2017** [reply]
Fettes **DANKE**

**Reply by tectumopticum** [reply]
@JohannMaierhofer dem Dank schließe ich mich an!

**Comment by JohannMaierhofer** [comment]
Ja, XRechnung 3.x ist ein reines XML Format für den Austauch B2B. Das is sicher nicht der Fokus in JVerein.

Was Hibiscus macht ist wohl ein hybrides Datenformat nach ZUGFeRD wo in einem PDF-Dokument (PDF/A-3) strukturierte Rechnungsdaten im XML-Format enthalten sind. Das XML Format könnte wohl XRechnung Version 2 sein. Das wäre schon etwas für JVerein weil diese Rechnung jeder als PDF lesen kann und dann aber direkt in seine Finanzsoftware enlesen kann.

Aber natürlich ist das kein Priorität Feature für JVerein. Ich wollte das nur hier anmelden. Vielleicht gibt es ja Entwickler die sich damit auskennen und gerne mal was für JVerein machen möchten.

**Comment by dippeal** [comment]
Für optierte Vereine wäre das bestimmt ein gutes feature. Als Beispiel fällt mir die Vermietung von Flächen ein.
